index.html

```diff
@@ -310,6 +319,224 @@ <h2>Contact Me</h2>
+        </div>
+        <div class="left1-block">
+          <h2 class="h2">
+            Uber Navigation
+          </h2>
+          <div class="frame4">
+            <p>FACEBOOK</p>
+              <img src="./images/Counter.png" alt="">
+            <p class="p1">FULL STACK DEV</p>
+              <img src="./images/Counter.png" alt="">
+            <p class="p1">2015</p>
+          </div>
+          <p class="primary2">
+            Experimental content creation feature that allows users to<br /> add to an existing story over the course of a day without<br /> spamming their friends.
+          </p>
+          <ul>
+            <li><a href="#">html</a></li>
+            <li><a href="#">Ruby on rails</a></li>
+            <li><a href="#">css</a></li>
+            <li><a href="#">javascript</a></li>
+          </ul>
+  <div class="action1">
+  <a href="#"><p>See Project</p></a>
+  </div>
+</div>
+</div>
+</section>
+
+<section class="about1"> 
+  <div>
+<div class="top">
+  <h2 class="h2">About<br />Myself</h2>
+  <div class="primary6">
+    <p>
+      Hello I am a software developer! I can help you build a product ,
+      feature or website Look through some of my work and experience! 
+      If you like what you see and have a project you need coded, do not hesitate to contact me.
+    </p>
+  </div>
+  <div class="connect1"> 
+   <a href="#"><p>LET US CONNECT</p></a>
+  </div>
+  <div class=" social_links1">
+    <ul>
+      <li>
+        <a href="#">
+          <img class="twit1-icon" src="./images/twitter.png" alt="twit-icon">
+        </a>
+      </li>
+      <li>
+        <a href="#">
+          <img class="git1-icon" src="./images/github.png" alt="git-icon">
+        </a>
+      </li>
+      
+      <li>
+        <a href="#">
+          <img class="ln1-icon" src="./images/Linkedin-icon.png" alt="ln-icon">
+        </a>
+      </li>
+      <li>
+        <a href="#">
+          <img class="m1-icon" src="./images/medium.png" alt="m-icon">
+        </a>
+      </li>
+      <li>
+        <a href="#">
+          <img class="a1-icon" src="./images/angellist.png" alt="a-icon">
+        </a>
+      </li>
+     </ul>
+    
+  </div>
+  <div class="action11">
+  <a><p>Get my Resume</p></a>
+  </div>
+</div>
+</div>
+<div></div>
+<div class="skill-list">
+  <div class="frame11">
+    <div class="language">
+      <span class="text">Languages</span>
+      <span class="down-angle"><i class="fa-solid fa-angle-down"></i></span>
+    </div>
+    <ul>
+      <li class="ls">
+        <div class="language-list">
+          <ul>
+            <li><a href="#"> <img src="./images/js.png" alt=""><span>javaScript</span> </a></li>
+            <li><a href="#"><img src="./images/html.png" alt=""><span>HTML</span></a></li>
+            <li><a href="#"><img src="./images/css.png" alt=""><span>CSS</span></a></li>
+          </ul>
+        </div>
+      </li>
+        <li class="my-work">
+          <div class="frameworks">
+            <span >Frameworks</span>
```

Comment:
### OPTIONAL 

- [ ] Kindly  wrap the two blocks  of about, & languages (frameworks and skills) in a section, this will be more semantic

- [ ] In this section you have two blocks; The left one takes 40% and the right one the remaining 60% of the page's width, and the contents of each of them are aligned to the right

- [ ] Kindly add the border  under the frameworks and  skills tags